spring-cloud-azure-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/AzureConfigPropertySource.java

```diff
@@ -5,24 +5,40 @@
  */
 package com.microsoft.azure.spring.cloud.config;
 
-import com.microsoft.azure.spring.cloud.config.domain.KeyValueItem;
-import com.microsoft.azure.spring.cloud.config.domain.QueryOptions;
-import org.springframework.core.env.EnumerablePropertySource;
-
+import java.io.IOException;
+import java.util.ArrayList;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.env.EnumerablePropertySource;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.spring.cloud.config.domain.KeyValueItem;
+import com.microsoft.azure.spring.cloud.config.domain.QueryOptions;
+import com.microsoft.azure.spring.cloud.config.feature.management.entity.Feature;
+import com.microsoft.azure.spring.cloud.config.feature.management.entity.FeatureManagementItem;
+import com.microsoft.azure.spring.cloud.config.feature.management.entity.FeatureSet;
+
 public class AzureConfigPropertySource extends EnumerablePropertySource<ConfigServiceOperations> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AzureConfigPropertySource.class);
     private final String context;
+
     private Map<String, Object> properties = new LinkedHashMap<>();
+
     private final String storeName;
+
     private final String label;
 
+    private ObjectMapper mapper = new ObjectMapper();
+
     public AzureConfigPropertySource(String context, ConfigServiceOperations operations, String storeName,
-                                     String label) {
-        // The context alone does not uniquely define a PropertySource, append storeName and label to uniquely
+            String label) {
+        // The context alone does not uniquely define a PropertySource, append storeName
+        // and label to uniquely
         // define a PropertySource
```

Comment:
nit: the comment breaks into 3 lines, but the 2nd and 3rd line can be in one line?